src/main/java/com/google/cloud/tools/app/impl/cloudsdk/CloudSdkAppEngineVersions.java

```diff
@@ -63,6 +63,7 @@ public void start(VersionsSelectionConfiguration configuration) throws AppEngine
     arguments.add("start");
     arguments.addAll(configuration.getVersions());
     arguments.addAll(GcloudArgs.get("service", configuration.getService()));
+    arguments.addAll(GcloudArgs.get("project", configuration.getProject()));
```

Comment:
Maybe we can add a method GcloudArgs.get(Configuration config) that will translate all the common configuration? This would eliminate the duplication of this line, and possibly more in the future.